ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh

```diff
@@ -97,4 +99,11 @@ cd "$(mktemp -d)"
 
 git clone --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
 make ci/prepare/e2e-branch
+# Register Pac host
+pac_host=$(kubectl -n openshift-pipelines get route pipelines-as-code-controller -o jsonpath='{.spec.host}')
+curl -k -X POST -H "Authorization: Bearer ${QE_SPRAYPROXY_TOKEN}" "https://${QE_SPRAYPROXY_HOST}/backends" --data "{\"url\": \"https://$pac_host\"}"
```

Comment:
> > /pj-rehearse pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e
> 
> @psturc can I move the registering/unregistering logic into [ci/test/e2e](https://github.com/redhat-appstudio/e2e-tests/blob/main/magefiles/magefile.go#L206) to avoid the unregistering won't happen if the tests (make ci/test/e2e) fails

yep this should be definitely put into installer in e2e-tests [after the bootstrap step](https://github.com/redhat-appstudio/e2e-tests/blob/e4581f0f971652126b57f4ab8fe7a0a728a24507/magefiles/magefile.go#L224)

Reply:
thank you. I made this PR as draft. when I implement it in e2e-tests , we can close this PR.

Reply:
well it just needs to be updated, we still need [these env vars](https://github.com/openshift/release/pull/42057/files#diff-275831e17826ca1203ee72a62de422ff411664d391d74122cf998655d773e602R34-R35) to be exported here

Reply:
right. I will update this PR